tests/integration/security/reachability/reachability_test.go

```diff
@@ -33,6 +33,31 @@ import (
 	"istio.io/istio/tests/integration/security/util/connection"
 )
 
+func createTestCases(from, to echo.Instance, ports []echo.Port, expectSuccess bool) []connection.Checker {
```

Comment:
Is this function used in this PR?

Reply:
Good catch. I intended to make it easy to generate test cases, but then decide to be explicit maybe is a better thing to read the test. Removed.